teams/codebender.md

```diff
@@ -19,14 +19,17 @@ permalink: /teams/codebender/
 
 ## Workplan
 
-* [ ] Project set up
-* [ ] Domain understood
+* [x] Domain understood - Week 1, Thu
+* [x] Product demo - Week 2, Mon
+* [x] Project set up - Week 2, Tue
 * [ ] Feature planned
 
 ## Contributions
  
 #### @thngkaiyuan (Thng Kai Yuan)
-* ...
+### Technology expertise in Information Security
+ * Sharing on Cross-Site Request Forgeries ([Blog Post on CSRF](http://cs3281-2.blogspot.sg/2016/01/cross-site-request-forgeries-csrf.html))
+ * Write-up on my participation in Insomni'hack 2016: ([Blog Post on the Smartcat Challenge](http://cs3281-2.blogspot.sg/2016/01/te-insomnihack-2016-teaser-smartcat-web.html))
```

Comment:
That's great! Do I simply add a one-liner "My participation in NUS GreyHats" or would it be better for me to list the activities that I do in there (e.g. Meetup with Cyber Test Systems to discuss cyber range scenarios, Weekly Training: [1], [2], [3])?